### PR TITLE
Ensure we use the same GRPC version everywhere

### PR DESCRIPTION
**Pull request by artur-ciocanu**
# Description

Ensure we use the same GRPC version everywhere

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #1136 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

**Comment by artur-ciocanu** [comment]
@artursouza and @cicoyle this another tiny PR that should make our life easier, since having two GRPC versions on the classpath can lead to "interesting" issues.

If everything looks good, please approve and merge.

Thank you 🙇

File: sdk-actors/src/test/java/io/dapr/actors/TestUtils.java [hunk 1]
```diff
@@ -47,13 +47,11 @@ public static void assertThrowsDaprException(
   public static <T extends Throwable> void assertThrowsDaprException(
       Class<T> expectedType,
       String expectedErrorCode,
-      String expectedErrorMessage,
```
Comment (salaboy) [inline sdk-actors/src/test/java/io/dapr/actors/TestUtils.java:50]: @artur-ciocanu why is this gone?

Comment (cicoyle) [reply sdk-actors/src/test/java/io/dapr/actors/TestUtils.java:50]: instead of deleting it can we fix it forward in a separate PR?

Comment (artur-ciocanu) [reply sdk-actors/src/test/java/io/dapr/actors/TestUtils.java]: Fixed.

**Comment by artur-ciocanu** [comment]
@salaboy @cicoyle @artursouza I have addressed the PR comments. Could you please take another look. Thank you!

**Review by salaboy — APPROVED** [review]
LGTM

**Review by cicoyle — APPROVED** [review]

**Review by artursouza — DISMISSED** [review]

**Review by cicoyle — APPROVED** [review]

**Comment by artur-ciocanu** [comment]
@artursouza @salaboy and @cicoyle the build is finally 🟢 could you please approve and merge.

Thank you! 🙇

**Review by cicoyle — APPROVED** [review]

**Review by artursouza — APPROVED** [review]

**Pull request merged**